WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -148,6 +153,9 @@ public void end() {
             }
             Map<String, Object> properties = getCommonProperties();
             properties.put(KEY_OUTCOME, mOutcome.toString().toLowerCase(Locale.ROOT));
+            if (capabilities != null) {
+                properties.put(KEY_CAN_VIEW_EDITOR_ONBOARDING, capabilities.getCanViewEditorOnboarding());
```

Comment:
Why do we need to send this property when the editor session ends? Won't it just always be `false`?

Reply:
> Why do we need to send this property when the editor session ends? 

The hope is that having this capability reported would allow us to observe any differences in user behavior between users with and without this capability enabled. Including the property on for `wpandroid_session_start` and `wpandroid_session_end` was recommended by @hypest. From my perspective, including the property on these two events does make sense, with them serving as "bookends" to other events that occur in the editor.  

> Won't it just always be `false`?

Currently, yes, it will always be `false`. However, I am actively refactoring how the capability is computed so that once a user has this capability enabled, it is always enabled. This involves separating the tracking for `hasLaunchedGutenbergEditor` to no longer be [a part of the capability computation](https://github.com/wordpress-mobile/WordPress-Android/blob/2814689c911c394bab847696dba6d3dfafa94d8b/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L2296-L2298). 

Knowing this context, do you think it makes sense to add this property to `wpandroid_session_end`?

Reply:
Makes sense to me 👍